client/client_test.go

```diff
@@ -3952,51 +3952,119 @@ func testMergeOp(t *testing.T, sb integration.Sandbox) {
 		require.NoError(t, err)
 
 		require.NoError(t, fstest.CheckDirectoryEqualWithApplier(destDir, fstest.Apply(files...)))
+
+		// verify the image exports+imports the same
+		var target string
+		var exports []ExportEntry
+		if os.Getenv("TEST_DOCKERD") == "1" {
```

Comment:
I guess these checks would be better with interface detection but looks like it is already used elsewhere so let's do that separately.